libs/platform/src/lib/components/table/components/table-p13-dialog/sorting/sorting.component.html

```diff
@@ -29,6 +28,7 @@
                 [value]="rule.columnKey"
                 (valueChange)="_onRuleColumnKeyChange(rule, $event)"
                 [compact]="true"
+                [ariaLabel]="'Rule #'+ (i + 1) + ', sort by'"
```

Comment:
should we add i18n here?

Reply:
I think yes to indicate what exactly this select for.

Reply:
then please add i18n for ariaLabel